index.bs

```diff
@@ -342,6 +347,24 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
 
 NOTE: [[!AV1]] supports encoding either 3-component images (whose semantics are given by the <code>matrix_coefficients</code> element), or 1-component images (monochrome). When an image requires a different number of components, multiple auxiliary images may be used, each providing additional component(s), according to the semantics of their <code>aux_type</code> field. In such case, the maximum number of components is restricted by number of possible items in a file, coded on 16 or 32 bits.
 
+<h3 id="derived-images">Derived Image Items</h3>
+
+<h4 id="tonemap-derivation">Tonemap Derived Image Item</h4>
+
+A tonemap derived image item ('tmap') as defined in [[!HEIF]] may be used in an [=AVIF=] file. <assert>When present, the base image item and the 'tmap' image item should be grouped together by an '[=AVIF/altr=]' entity group as recommended in [[!HEIF]].</assert>
+
+<h2 id="groups">Entity groups</h2>
+
+The GroupsListBox ('<dfn for="AVIF">grpl</dfn>') defined in [[!ISOBMFF]] may be used to group multiple image items in a file together. The type of the group describes how the image items are related.
```

Comment:
I was debating whether we want to have language here saying that AVIF parsers are expected to understand the groups listed below or something saying that other group types than 'altr' and 'ster' may be present. But in the end I couldn't think of concise language to say it and simply skipped it.

Reply:
The group type is the type of the enclosed box, and I think it's a general rule in isobmff that unknown box types should be ignored, so there should be no need to say anything.
In my opinion it doesn't hurt to repeat it anyway (i.e. "Decoders should ignore unknown group types", which implies that types other than altr and ster may be present). But since it's not done anywhere else in this spec, it might be confusing to only say it here?

Reply:
Added `<assert>Decoders should ignore groups of unknown type.</assert>`